odoo/exceptions.py

```diff
@@ -126,3 +126,24 @@ class ValidationError(UserError):
 
         When you try to create a new user with a login which already exist in the db.
     """
+
+
+
+# Deprecated exceptions, only kept for backward compatibility, may be
+# removed in the future *without* any futher notice than the Deprecation
+# Warning.
+
+class except_orm(UserError):
+    def __init__(self, *args, **kwargs):
+        warnings.warn("except_orm is a deprecated alias to UserError.", DeprecationWarning)
+        super().__init__(*args, **kwargs)
+
+class Warning(UserError):
```

Comment:
The class `Warning` is defined twice in this file.
Remove the other one (I prefer this more concise version).